mail_view_builder compiler error

value used after move

```
error[E0382]: use of moved value: `mail_view_builder`
   --> packages/target-tauri/src-tauri/src/html_window/mod.rs:204:29
    |
130 |     let mut mail_view_builder = tauri::webview::WebviewBuilder::new(
    |         --------------------- move occurs because `mail_view_builder` has type `tauri::WebviewBuilder<tauri_runtime_wry::Wry<tauri::EventLoopMessage>>`, which does not implement the `Copy` trait
...
200 |         let data_dir = mail_view_builder.data_directory(tmp_data_dir.clone());
    |                        ----------------- value moved here
...
204 |         mail_view_builder = mail_view_builder.incognito(true);
    |                             ^^^^^^^^^^^^^^^^^ value used here after move

```

--- a/packages/target-tauri/src-tauri/src/html_window/mod.rs
+++ b/packages/target-tauri/src-tauri/src/html_window/mod.rs
@@ -197,7 +197,7 @@ pub(crate) fn open_html_window(
     let tmp_data_dir = get_temp_folder_path(&app)?.join(uuid::Uuid::new_v4().to_string());
     #[cfg(not(target_vendor = "apple"))]
     {
-        let data_dir = mail_view_builder.data_directory(tmp_data_dir.clone());
+        mail_view_builder = mail_view_builder.data_directory(tmp_data_dir.clone());
     }
     #[cfg(not(target_os = "android"))]
     {

--- a/packages/target-tauri/src-tauri/tauri.conf.json5
+++ b/packages/target-tauri/src-tauri/tauri.conf.json5
@@ -26,10 +26,10 @@
         "style-src": "'self' 'unsafe-inline'",
         "font-src": "'self'",
         "script-src": "'self' 'wasm-unsafe-eval'",
-        "worker-src": "blob: ",
-        "child-src": "blob: ",
-        "img-src": "'self' data: blob: dcblob: webxdc-icon: ",
-        "media-src": "'self' dcblob:"
+        "worker-src": "blob: http://blob.localhost",
+        "child-src": "blob: http://blob.localhost",
+        "img-src": "'self' data: blob: http://blob.localhost dcblob: http://dcblob.localhost webxdc-icon: http://webxdc-icon.localhost",
+        "media-src": "'self' dcblob: http://dcblob.localhost"
       },
       // TODO postponed to later
       // "pattern": {

